src/kademlia/get_item.cpp

```diff
@@ -51,6 +51,9 @@ void get_item::got_data(bdecode_node const& v,
 {
 	// we received data!
 
+	// for put_immutable_item, there is no data_callback.
+	if (!m_data_callback) return;
```

Comment:
why do we need to skip this function here? did there used to be a data callback for put_immutable_item() or was this just an unrelated bug with no test coverage?